storage/revision.go

```diff
@@ -21,9 +21,15 @@ import "encoding/binary"
 // is a '_'. The last 8 bytes is the revision.sub in big-endian format.
 const revBytesLen = 8 + 1 + 8
 
+// revision is an index for key/value operations,
+// useful for tracking versions of keys.
 type revision struct {
+	// main revision increases whenever request is applied.
```

Comment:
this is not accurate. main is the main revision of a set of changes that happens atomically.